README.md

```diff
@@ -1,29 +1,32 @@
-# PyClustree
+# opticlust
 
-Single cell clustering at a glance. 
+Single cell clustering and recommendation at a glance. 
```

Comment:
recommendation or recommendation**s**?

Reply:
The latter is perhaps better if we want to recommend more than a single clustering resolution. I was thinking of returning one for "low", "medium" and "high" resolutions.